test/integ/pandas/test_pandas_tools.py

```diff
@@ -417,6 +417,54 @@ def test_write_pandas_create_temp_table_deprecation_warning(
             cnx.execute_string(drop_sql)
 
 
+@pytest.mark.parametrize("use_logical_type", [None, True, False])
```

Comment:
Could we add unit tests for for all possible combinations and only run a single integration test?

Reply:
use_logical_type is used to create sql_use_logical_type in write_pandas, which is directly passed to a SQL command (line 420 of pandas_tools.py). Testing only 1 parameter value and mocking the other 2 values, would therefore not cover the entire implementation.

Reply:
> Testing only 1 parameter value and mocking the other 2 values, would therefore not cover the entire implementation.

Right, unit testing would ensure that we have correctness and cover everything, but if we had an integration test for each of the `write_pandas` flags then the tests would run for twice as long at least.